pkg/cmd/operator/cmd.go

```diff
@@ -113,6 +120,11 @@ func NewOperator() *cobra.Command {
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 
+			caConfigMapPath := filepath.Join(caConfigMapMountPath, caConfigMapName)
+			if _, err := os.Stat(caConfigMapPath); err != nil {
+				go terminateWhenProxyChanges(caConfigMapPath, cancel)
+			}
```

Comment:
I think the place where I am calling the goroutine is not correct - any pointers?